analyzer/src/funTest/kotlin/managers/PipFunTest.kt

```diff
@@ -99,6 +99,21 @@ class PipFunTest : WordSpec() {
 
                 result.toYaml() shouldBe expectedResult
             }
+
+            "capture metadata using python-inspector" {
```

Comment:
Does this test add anything compared to the other tests from the same class? Because python-inspector is now used for the other tests as well.

Reply:
It would be good to have a test scenario that cannot be handled by the original pip-based implementation, but is correctly resolved by python-inspector.

Reply:
And we are updating the test to represent something different and uniquely handled by this new tool